src/ui/otb/newOtbGame.tsx

```diff
@@ -33,11 +37,30 @@ export default {
       isOpen(false)
     }
 
+    function addStage () {
```

Comment:
Can't we reuse those function that must be defined in `../shared/clock`?

Reply:
Yes, good idea. Done.